feder/main/management/commands/createsuperuserwithpassword.py

```diff
@@ -0,0 +1,33 @@
+# credit: https://stackoverflow.com/a/42491469
+from django.contrib.auth.management.commands import createsuperuser
+from django.core.management import CommandError
+
+
+class Command(createsuperuser.Command):
```

Comment:
Proponuje w tym pliku zamieścić informacje, że komenda jest wymagana wyłącznie w Django <3.0, abyśmy przy sprzątaniu rzeczy poniżej wersji 3.0 wiedzieli, że można to bezpiecznie usunąć. Przyjdzie czas, że odetniemy się od wszystkiego poniżej 3.0.